nodejs12.x/cookiecutter-typescript-app-template/{{cookiecutter.project_name}}/app/Makefile

```diff
@@ -1,17 +1,27 @@
 build-getByIdFunction:
-	# `ci` instead of `install` since node_modules will have broken symlinks when SAM CLI copies it into the build folder
-	npm ci
+	npm install
+# workaround https://github.com/aws/aws-sam-cli/issues/2565
```

Comment:
whitespace is very important in Makefiles, this should be indented by tab

Reply:
i think unindented is actually correct here because otherwise windows shells will try to interpret the line as an command (which only works in powershell i think)

Reply:
Richard is correct. My attempts to run previous indented versions would try to run the comment as a command.